src/main/java/uk/gov/pay/api/model/links/Link.java

```diff
@@ -44,4 +46,19 @@ public String toString() {
                 ", method='" + method + '\'' +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
might it not be easier to use https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/builder/EqualsBuilder.html?

Reply:
It’s what IntelliJ automatically generated

Reply:
ok then, in that case :)